migrations/2015_01_20_181911_create_event_store_table.php

```diff
@@ -12,14 +12,22 @@ class CreateEventStoreTable extends Migration
      */
     private $eventStoreTableName;
 
+    private $eventStoreConnection;
+
     public function __construct()
     {
         $this->eventStoreTableName = Config::get('broadway.event-store-table', 'event_store');
+        $this->eventStoreConnection = Config::get('broadway.event-store.connection', 'default');
+
+        if ($this->eventStoreConnection === 'default') {
+            $config = app(\Illuminate\Config\Repository::class);
+            $this->eventStoreConnection = $config->get('database.default');
+        }
```

Comment:
If the event store connection is default you can nullify the parameter to achieve the same effect. That way you don't need to resolve the name from the config.

Reply:
If you leave only 

```
$this->eventStoreConnection = Config::get('broadway.event-store.connection');
```

the code is broken In Dbal.php line 55:

  Undefined index: username

because are null parameters connection, for that reason I have managed the default

Reply:
For clarify, this is what I meant.

```suggestion
        if ($this->eventStoreConnection === 'default') {
            $this->eventStoreConnection = null;
        }
```

Reply:
Same error of undefined index: username

Reply:
Any news on this @Lavoaster ?